frb_codegen/src/config.rs

```diff
@@ -62,6 +62,9 @@ pub struct RawOpts {
     /// Show debug messages.
     #[structopt(short, long)]
     pub verbose: bool,
+    /// Skip dependencies check.
+    #[structopt(long)]
+    pub skip_deps_check: bool,
```

Comment:
or whichever `short` / `long` arg name you prefer @fzyzcjy

Reply:
LGTM, let's just make it long-name-only currently, since it may not be frequently used so short name may not be needed